src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy

```diff
@@ -602,6 +640,47 @@ class GradleProjectPlugin implements Plugin<Project> {
         configuration.resolvedConfiguration.rethrowFailure()
     }
 
+    static void updateVersionModuleAligns(String group, String module, String version) {
+        String inputModule = "$group:$module"
+
+        if (MODULES_MINIMUMS_TO_TRACK[inputModule]) {
+            def curOverrideVersion = versionModuleAligns[inputModule]
+            if (curOverrideVersion) {
+                curOverrideVersion['version'] =
+                    acceptedOrIntersectVersion(
+                        version,
+                        curOverrideVersion['version'] as String
+                    )
+            }
+            else
+                versionModuleAligns[inputModule] = [version: version]
+        }
+
+
+        def rule = MODULE_DEPENDENCY_MINIMUMS[inputModule]
+
+        rule.each {
```

Comment:
Discussed with Josh and decided to switch from using `it`to explicitly defining the key/value loop parameters to make it a bit clearer and more consistent